strings/accent_universal.json

```diff
@@ -34,15 +34,15 @@
 		"jehovah": "zizo",
 		"retarded": "foolish",
 		"retard": "nimrod",
-		"negro": "gnerd",
```

Comment:
Keep these. Changing them to Ebon is even more racist and will lead to people purposefully typing out these slurs.